src/CTA.WebForms2Blazor/Factories/FileInformationFactory.cs

```diff
@@ -6,9 +6,9 @@
 
 namespace CTA.WebForms2Blazor.Factories
 {
-    public static class FileInformationFactory
+    public class FileInformationFactory
     {
-        public static FileInformation Build(FileInfo document)
+        public FileInformation Build(FileInfo document, string sourceProjectPath)
```

Comment:
`Build()` method can likely be made static since it does not rely on any variables within the `FileInformationFactory` class itself.

Reply:
Regarding the decision to make the FileInformationFactory class non static vs static, Emery and I had a conversation about how it might be useful in the future to initialize different information in the FileInformationFactory instances and thus non static but I'll go over it again and see if this is really necessary

Reply:
Sounds good. No need to revisit if you guys have already discussed this point.

Reply:
To add to this, I think sourceProjectPath will be the same whenever the Build (and BuildMany) function is used so I think it might be best to move this into the constructor of the factory, assign it to a private variable, then use that when calculating the relative path to create FileInformation objects